dsls/sql/src/test/java/org/apache/beam/dsls/sql/mock/MockedUnboundedTable.java

```diff
@@ -38,14 +39,16 @@
  * A mocked unbounded table.
  */
 public class MockedUnboundedTable extends MockedTable {
-  private List<Pair<Duration, List<BeamSqlRow>>> timestampedRows = new ArrayList<>();
+  /** rows flow out from this table with the specified watermark instant. */
+  private final List<Pair<Duration, List<BeamSqlRow>>> timestampedRows = new ArrayList<>();
```

Comment:
Actually I did it in purpose, Let's look at some examples. First example:

```java
     addRows(
            Duration.ZERO,
            1, 1, 1, FIRST_DATE,
            1, 2, 2, FIRST_DATE
        )
```

It means these data are added after the specified watermark(ZERO) -- just after the first window starts. Another example:

```java
        .addRows(
            WINDOW_SIZE.minus(Duration.standardSeconds(1)),
            2, 2, 3, SECOND_DATE,
            2, 3, 3, SECOND_DATE
        )
```

It means these data are emitted just before the first window close. One more example:


```java
        .addRows(
            WINDOW_SIZE.plus(Duration.standardSeconds(1)),
            2, 2, 3, SECOND_DATE,
            2, 3, 3, SECOND_DATE
        )
```

It means these data are emitted just after the first window close, and after the second window starts.

Very handy and easy to use, right :)